### DISCUSSION TITLE
Is it possible to have an <sp-action-menu> use an <sp-avatar> for its icon?

### DISCUSSION THREAD

**Discussion by InfinityMin3r** [adobe/spectrum-web-components#3949] [Q&A]
In the Spectrum design documentation, they suggest that an optional avatar can be placed on the far right and that it should, when clicked, open a menu with actions related to logging in/out or signing up.

I have tried to use the <sp-action-menu> for this, but when I attempt to use <sp-avatar> as an icon, the placement of the avatar looks very janky. It isn't clear to me if the <sp-action-menu> was ever meant to support this.

Is there an easy way to utilize the existing components for this? Or is the best tack to try and use java/type script to have the avatar toggle a popover menu when clicked?

**Comment by Westbrook** [comment]
An `<sp-avatar>` manages its own size, whereas `<sp-icon>` (and derivative patters) adopt size from their host. This means that while you can use an `<sp-avatar>` within an `<sp-action-menu>`, you need to specifically choose to use the `size="75"` variant of the `<sp-avatar>`, like [this demo](https://studio.webcomponents.dev/edit/EYQn9T6wOnieZbv4xnPj/src/index.ts?branch=avatar%40wfVpXdsTYhf9mY2slxW4t2Y2SjC3&p=stories).
